racket/src/regexp/parse/unicode.rkt

```diff
@@ -67,7 +73,8 @@
                            "unrecognized property name in `\\~a{}`: `~a`"
                            (integer->char p-c)
                            (list->string (map integer->char l)))]))
-    (values (rx:unicode-categories categories (= p-c (char->integer #\p)))
+    (define prop-negated? (= p-c (char->integer #\p)))
```

Comment:
Er, this is clearly misnamed, since it's true when the property is _not_ negated. And if it were correctly named, `xor` would be the wrong truth function. (I'd want the complement instead.)
So the current code works but is confusing. I'll change that...

Reply:
Better now.